contracts/modules/STO/Capped/CappedSTO.sol

```diff
@@ -197,24 +194,10 @@ contract CappedSTO is CappedSTOStorage, STO, ReentrancyGuard {
     function _preValidatePurchase(address _beneficiary, uint256 _investedAmount) internal view {
         require(_beneficiary != address(0), "Beneficiary address should not be 0x");
         require(_investedAmount != 0, "Amount invested should not be equal to 0");
-        uint256 tokens;
-        (tokens, ) = _getTokenAmount(_investedAmount);
-        require(totalTokensSold.add(tokens) <= cap, "Investment more than cap is not allowed");
         /*solium-disable-next-line security/no-block-members*/
         require(now >= startTime && now <= endTime, "Offering is closed/Not yet started");
     }
 
-    /**
-    * @notice Validation of an executed purchase.
-      Observe state and use revert statements to undo rollback when valid conditions are not met.
-    */
-    function _postValidatePurchase(
-        address _beneficiary,
-        uint256 /*_investedAmount*/
-    ) internal view {
-        require(_canBuy(_beneficiary), "Unauthorized");
```

Comment:
Did this logic go elsewhere?

Reply:
Nope, I missed it. Thanks for pointing out, I'll fix it.